nix/pkgs/libspdk/default.nix

```diff
@@ -42,70 +40,84 @@ stdenv.mkDerivation rec {
     python
   ] ++ stdenv.lib.optionals enableDebug [ cunit lcov ];
 
-  CONFIGURE_OPTS = ''
-    ${enableFeature enableDebug "debug"}
-    --without-isal --with-iscsi-initiator
-    --with-internal-vhost-lib --disable-tests --with-dpdk-machine=native
-    --with-crypto
-    --with-uring
-  '';
+  #${enableFeature enableDebug "unit-tests"}
```

Comment:
do we need this comment?

Reply:
For the upcoming releases the have split out some of the tests in 'test' and 'unit-test' that's not in our tree yet, I did try it out so I left it here as comment so that I don't forget to add it back later, I will make the comment more explicit 👍